gatsby/src/components/Interaction/Interaction.tsx

```diff
@@ -0,0 +1,55 @@
+import { Link } from "gatsby";
+import React from "react";
+
+import styles from "./Interaction.module.scss";
+
+export type InteractionProps = {
+	interactant: {
+		title: string;
+		drug: {
+			slug: string;
+		};
+	};
+	messages: {
+		additiveEffect: boolean;
+		evidence: string | null;
+		message: string;
+		severity: string;
+		severityOrder: number;
+	}[];
+};
+
+export const Interaction: React.FC<InteractionProps> = ({
+	interactant,
+	messages,
+}: InteractionProps) => {
+	return (
+		<>
+			{interactant.drug?.slug ? (
```

Comment:
I'd maybe just put this condition around the anchor/title to avoid duplicating the h3?

Reply:
Ta, done